strings/src/main/java/com/revenuecat/purchases/strings/Emojis.kt

```diff
@@ -0,0 +1,13 @@
+package com.revenuecat.purchases.strings
+
+object Emojis {
+   const val APPLE = "🍎"
+   const val ERROR = "\u203C\uFE0F"
```

Comment:
could we add a comment describing the emojis that are added directly as unicode?

Reply:
Yup, I can add that. The mix of emojis & unicode was due to the length of the unicode varying from length 4 and 5, which effects the way it's displayed.